designer/server/src/routes/forms/create.js

```diff
@@ -295,7 +250,56 @@ function redirectToTitleWithErrors(request, h) {
   })
 
   // Redirect POST to GET without resubmit on back button
-  return h.redirect('/create/title').code(303).takeover()
+  return h
+    .redirect(ROUTE_PATH_CREATE_TITLE)
+    .code(StatusCodes.SEE_OTHER)
+    .takeover()
+}
+
+/**
+ * @param {Request} request
+ * @param {ResponseToolkit} h
+ * @param {Error} [error]
+ */
+export function redirectToStepWithErrors(request, h, error) {
+  return redirectWithErrors(request, h, error, true)
+}
```

Comment:
Do we need this wrapper function?

It's the only place we call `redirectWithErrors()` and the `redirectToPreviousStep` param never changes

Reply:
It's used from `edit.js` as well.

In `create.js`, redirectWithErrors is called with `true`.
In `edit.js`, `redirectWithErrors` is called with `false`.